src/components/InfoBar/details.tsx

```diff
@@ -116,11 +116,11 @@ export const ColorStatus = styled.span`
 const truncateBalanceto5charsWhenNoDecimals = (value: string | number | undefined | null) => {
   try {
     if (value && BigInt(value)) {
+      if (typeof value === 'string') value = parseInt(value);
       if (BigInt(value) > BigInt(1e9)) {
         return '1e9+';
       } else if (BigInt(value) >= BigInt(1e6)) {
```

Comment:
This function could use some love for it's formatting.
Also if you 'return' from a path you should omit the `else` in the alternative branch.
will increase readability